orm/orm.go

```diff
@@ -335,9 +335,7 @@ func (o *orm) Insert(md interface{}) (int64, error) {
 	}
 	o.setPk(mi, ind, id)
 	if hookFunc, ok := o.afterInsert[name]; ok {
-		go func(md interface{}) {
-			hookFunc(md)
-		}(md)
+		return id, hookFunc(md)
```

Comment:
still thought there are no need for 'after' hooks return error or panic. 'after' hook's error can handled by developers, orm just need to trigger the hooks in goroutine mode.
better this way:
```
go func(md interface{}) {
    defer func() {
        if r := recover(); r != nil {
            fmt.Errorf("<Ormer.hook> hook AfterInsert error: %s", r)
        }
    }()
    hookFunc(md)
}(md)
```